scripts/superdesk-search/search.js

```diff
@@ -2659,7 +2659,14 @@
                          */
                         function getQuery() {
                             var metas = [];
+
                             angular.forEach(scope.meta, function(val, key) {
+                                //checkbox boolean values.
+                                if (typeof(val) === 'boolean') {
+                                    val = Number(val);
```

Comment:
convertBooleanToBinary sounds good ;) It's small PR so comments are also on *microlevel* ;)